===================================================================
RCS file: /cvs/gnome/gnomeweb-wml/docbook_html_style/db2html.xsl,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- db2html.xsl	2005/06/13 11:30:04	1.1
+++ db2html.xsl	2006/04/15 21:07:32	1.2
@@ -236,10 +236,12 @@
 				<xsl:with-param name="prev" select="false()"/>
 				<xsl:with-param name="next" select="$next"/>
 			</xsl:call-template>
-			<xsl:call-template name="yelp.title"/>
-			<xsl:apply-templates select="yelp:get-content(.)"/>
 			<div class="abstract">
-				<xsl:apply-templates select="/article/articleinfo/abstract"/>
+				<xsl:apply-templates select="/book/bookinfo/abstract"/>
+			</div>
+			<xsl:apply-templates select="yelp:get-content(.)"/>
+			<div class="titlepage">
+				<xsl:call-template name="book.titlepage"/>
 			</div>
 			<xsl:call-template name="yelp.navbar.bottom">
 				<xsl:with-param name="node" select="."/>
@@ -276,7 +278,7 @@
 				<xsl:attribute name="href">
 					<xsl:call-template name="yelp.toc.ref"/>
 				</xsl:attribute>
-				<xsl:value-of select="yelp:get-title-text(/article/articleinfo/abstract)"/>
+				<xsl:value-of select="yelp:get-title-text(/book)"/>
 			</a></td>
 		</xsl:if>
 		<xsl:call-template name="yelp.navbar.cell">
